seg_utils/src/label_main.py

```diff
@@ -47,9 +55,10 @@ def __init__(self):
 
         # color stuff
         self._num_colors = 25  # number of colors
-        self.colorMap = None
+        self.colorMap = None  # type: list
 
-        self._FD_Dir = '/home/nico/isys/data'  # QDir.homePath()
+        # self._FD_Dir = '/home/nico/isys/data'  # QDir.homePath()
+        self._FD_Dir = '/Users/jakob/coding/MedicalAnnotationFramework/seg_utils/examples'
```

Comment:
can you change temporary in-development variables like this to use the relative path?